pkgs/servers/apache-airflow/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, fetchFromGitHub
+, fetchPypi
+, python3
+}:
+
+let
+  python = python3.override {
+    packageOverrides = pySelf: pySuper: {
+      # flask-appbuilder doesn't work with sqlalchemy 2.x, flask-appbuilder 3.x
+      # https://github.com/dpgaspar/Flask-AppBuilder/issues/2038
+      flask-appbuilder = pySuper.flask-appbuilder.overridePythonAttrs (o: {
+        meta.broken = false;
+      });
+      flask-sqlalchemy = pySuper.flask-sqlalchemy.overridePythonAttrs (o: {
```

Comment:
Maybe add a comment as to why flask-sqlalchemy is being overridden specifically (due to the sqlalchemy dependency), even though it's pretty obvious I guess.